### ISSUE TITLE
Melhorias na ficha D&D 5e

### ISSUE THREAD

**Issue by Painisio** [rrpgfirecast/firecast#107]
Adicionar um botão na descrição da magia para enviar o texto direto para a mesa!

Já pensaram em usar a data base do 5e tools? Digo dar aos jogadores a possibilidade de importar as informações direto do site tendo em vista que eles disponibilizam até pros seus concorrentes... (roll20)

[https://5e.tools/index.html](url)
![Captura de tela 2023-06-03 131307](https://github.com/rrpgfirecast/firecast/assets/92547400/0c59f8e0-1fb1-4709-9c2c-adf41ae5004b)

**Comment by Painisio** [comment]
Meu amigo tem reclamado desse problema:

![Imagem do WhatsApp de 2023-06-01 à(s) 19 24 22](https://github.com/rrpgfirecast/firecast/assets/92547400/efbd531c-6606-454b-8331-21d82175a64f)

Ele não consegue editar os campos de ataque dos equipamentos visto que não aparece pra ele!

**Comment by Painisio** [comment]
@aislandener @TyrasNistru @Ruteski @VinnyAmbesek

**Comment by TyrasNistru** [comment]
O 2 é permitido contanto que o conteúdo seja apenas o do SRD.

**Comment by VinnyAmbesek** [comment]
> O 2 é permitido contanto que o conteúdo seja apenas o do SRD.

Sim, mas esse site em questão não usa apenas o que ta na ogl. Até tem dizendo embaixo pra só usar material de livros q vc comprou. De toda forma, a ogl da wizard nao ta mt confiavel.

**Comment by VinnyAmbesek** [comment]
@Painisio 
Acabei de subir uma atualização.
